init.php

```diff
@@ -30,6 +30,10 @@
     exit;
 }
 
+if (defined('PHP_UNIT_TEST')) { /* stop warnings about headers having already been sent */
```

Comment:
you could detect that the SAPI is CLI and avoid this ?

Reply:
Yeah, I also saw there was some ::isWeb() function somewhere .... 

I did think about doing phpsapiname() == 'cli' .... I don't know enough about the codebase to have an opinion on which is better.